iothub/device/src/Transport/RetryDelegatingHandler.cs

```diff
@@ -719,12 +719,6 @@ private void HandleConnectionStatusExceptions(IotHubException hubException)
             {
                 status = ConnectionStatusChangeReason.Bad_Credential;
             }
-            else if (hubException is DeviceDisabledException)
```

Comment:
There was a pending TODO to remove this exception mapping

Reply:
This was a specific ask from mcvp. Can you double check if this is not used by them?

Reply:
Sure, I will check with them.